### PR TITLE
Signs replacement

### LINKED ISSUE #27: Labels on the model should autocorrect from ">=" and "<=" to their unicode

### PR DESCRIPTION
**Pull request by EmilieSonne**
Closes #27

### CODE REVIEW

**Review by Dolmer1 — APPROVED** [review]
u have my approval. god weekend.

File: src/test/java/ecdar/utility/StringHelperTest.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+public class StringHelperTest {
+    @Test
+    public void ConvertSymbolsToUnicodeTest() {
+        String stringToReplace = "2 >= 4";
+        String result = StringHelper.ConvertSymbolsToUnicode(stringToReplace);
+        String expected = "2 \u2265 4";
+
+        assertEquals(expected, result);
+    }
```
Comment (mads256h) [inline src/test/java/ecdar/utility/StringHelperTest.java:26]: I don't understand why you have two test files that does the same tests

File: src/test/java/ecdar/utility/StringHelperTestSymbolToUnicode.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                {"2 <= 4", "2 \u2264 4"},
+                {"2 == 4", "2 == 4"},
+                {"2 > 4", "2 > 4"},
+                {"2 < 4", "2 < 4"}
+        });
+    }
+
+    @Test
+    public void testStringHelperConversionToUnicode() {
+        System.out.println(inputString + " bliver sammenlignet med " + expectedResult);
```
Comment (mads256h) [inline src/test/java/ecdar/utility/StringHelperTestSymbolToUnicode.java:40]: I think you forgot to delete this line.
You should not print anything in test cases.

File: src/test/java/ecdar/utility/TestRunner.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        System.out.println("Result of test StringHelperTestSymbolToUnicode: " + result1.wasSuccessful());
+
+        // ------------ Test 2: unicode to symbol ------------------
+        Result result2 = JUnitCore.runClasses(StringHelperTestUnicodeToSymbol.class);
+        for (Failure failure : result2.getFailures()) {
+            System.out.println(failure.toString());
+        }
+        System.out.println("Result of test StringHelperTestUnicodeToSymbol: " + result2.wasSuccessful());
+    }
+}
```
Comment (mads256h) [inline src/test/java/ecdar/utility/TestRunner.java:23]: Im pretty sure the IDE/gradle supports running tests.
I really don't think you should have a TestRunner

**Review by mads256h — CHANGES_REQUESTED** [review]
Small changes needed otherwise lgtm

File: build.gradle [hunk 4]
```diff
@@ -66,6 +66,7 @@ dependencies {
     //GRPC:
     implementation "io.grpc:grpc-protobuf:${grpcVersion}"
     implementation "io.grpc:grpc-stub:${grpcVersion}"
+    testImplementation 'junit:junit:4.13.1'
```
Comment (mads256h) [inline build.gradle:69]: The project already has tests why is this needed?

Comment (EmilieSonne) [reply build.gradle:69]: to function

File: src/test/java/ecdar/utility/TestRunner.java (hunk 3 above)
Comment (mads256h) [reply src/test/java/ecdar/utility/TestRunner.java:23]: Hmm that does not feel right. We had parameterized tests in java last semester so it should be possible.

File: src/test/java/ecdar/utility/StringHelperTest.java (hunk 1 above)
Comment (EmilieSonne) [reply src/test/java/ecdar/utility/StringHelperTest.java]: +

File: src/test/java/ecdar/utility/TestRunner.java (hunk 3 above)
Comment (EmilieSonne) [reply src/test/java/ecdar/utility/TestRunner.java:23]: https://www.tutorialspoint.com/junit/junit_parameterized_test.htm

Comment (mads256h) [reply src/test/java/ecdar/utility/TestRunner.java:23]: Jeg giver det et forsøg, men kører TestRunner overhovedet hvis man kører `gradlew test`?

Comment (mads256h) [reply src/test/java/ecdar/utility/TestRunner.java]: Fixed!

**Review by mads256h — APPROVED** [review]
LGTM!

**Review by mads256h — APPROVED** [review]
